src/main/scala/tech/cryptonomic/conseil/generic/chain/MetadataTypes.scala

```diff
@@ -1,22 +1,25 @@
-package tech.cryptonomic.conseil.tezos
+package tech.cryptonomic.conseil.generic.chain
 
 import com.fasterxml.jackson.core.`type`.TypeReference
 import com.fasterxml.jackson.module.scala.JsonScalaEnumeration
 import slick.ast.{OptionType, Type}
-import tech.cryptonomic.conseil.tezos.PlatformDiscoveryTypes.DataType.DataType
-import tech.cryptonomic.conseil.tezos.PlatformDiscoveryTypes.KeyType.KeyType
+import tech.cryptonomic.conseil.generic.chain.MetadataTypes.DataType.DataType
+import tech.cryptonomic.conseil.generic.chain.MetadataTypes.KeyType.KeyType
 
 /**
   * Classes used for Platform routes
   */
-object PlatformDiscoveryTypes {
+object MetadataTypes {
 
   /** Class required for DataType enum serialization */
   class DataTypeRef extends TypeReference[DataType.type]
 
   /** Class required for KeyType enum serialization */
   class KeyTypeRef extends TypeReference[KeyType.type]
 
+  /** Case class representing network */
```

Comment:
Now that typed configurations are available with specific classes to hold Platform data, we can check out if `tech.cryptonomic.conseil.config.Platforms` could be reused for some of this metadata-types by chance

Reply:
I wouldn't want to mix config types with types returned for the end user

Reply:
My bad, I didn't meant "reuse" literally, but to extract the configuration information to build the returned objects

Reply:
I'm using new config for getting those objects - you mean that?